### DISCUSSION TITLE
How to get the generated documents in the global or home page, or by category?

### DISCUSSION THREAD

**Discussion by cliek** [vuepress/core#146] [Q&A] [answered]
Hello, I checked the composition API related document [client API](https://vuepress2.netlify.app/zh/reference/client-api.html#composition-api)Among them, I didn't find an API to get the generated MD documents. 
For example, I need to go to the home page or the category page or the corresponding number of documents, or I need to jump to the page number of the current category page, These functions have been implemented in V1 based on [@vuepress/theme-default? Client API](https://vuepress-plugin-blog.ulivz.com/client-api/#pagination)But for V2, I don't know how to solve it yet?

If this needs to be implemented by theme developers themselves, can you recommend the implementation idea, because V2 has not mentioned the process of how to generate documents and how to generate documents by classification.

Thank you very much!

**Comment by Mister-Hope** [comment]
All above you mentioned came from `@vuepress/plugin-blog`, not vuepress core or default theme. And you can subscribe #54. I think plugin-blog may be something focused next stage.

**Comment by Mister-Hope** [comment, marked as answer]
Updated: `@vuepress/core` supports routeMeta for you to inject meta infomation to routes. And `vuepress-plugin-blog2` provides basic blog features.

So there should not be any problems with your need.
